pkg/querier/queryrange/codec.go

```diff
@@ -909,3 +895,31 @@ func NewEmptyResponse(r queryrangebase.Request) (queryrangebase.Response, error)
 		return nil, fmt.Errorf("unsupported request type %T", req)
 	}
 }
+
+func mergeLokiResponse(responses ...queryrangebase.Response) *LokiResponse {
+	var (
+		lokiRes       = responses[0].(*LokiResponse)
```

Comment:
This is just refactoring from a code used else where. In our usage we always pass items. In the previous usage they already checked that before, I'll add a quick check just in case.